src/Mvc/Mvc.NewtonsoftJson/test/NewtonsoftJsonOutputFormatterTest.cs

```diff
@@ -549,6 +549,24 @@ async IAsyncEnumerable<int> AsyncEnumerableThrows([EnumeratorCancellation] Cance
         }
     }
 
+    [Fact]
+    public void JsonSerializerSettingsCopyConstructor_DoesNotOverrideDateFormatHandling()
+    {
+        // Arrange
+        var originalSettings = new JsonSerializerSettings
+        {
+            DateFormatHandling = DateFormatHandling.MicrosoftDateFormat,
+            DateFormatString = null
+        };
+
+        // Act
+        var copiedSettings = new JsonSerializerSettings(originalSettings);
```

Comment:
Just so you know, in this particular case, I would have accepted the change without a test, but I appreciate the effort. This is only testing the behavior of JsonSerializerSettings and not the CreateJsonSerializer method changed by this PR. It really belongs more in the https://github.com/JamesNK/Newtonsoft.Json repo.

Reply:
Thanks, I'll keep that in mind for any future contributions